packages/kit/src/runtime/app/images.js

```diff
@@ -0,0 +1,47 @@
+import { DEV } from 'esm-env';
+import { sizes, loader, domains } from '__sveltekit/images';
+
+/**
+ * @param {string} src
+ * @param {any} [options]
+ * @returns {{ src: string, srcset?: string }}
+ */
+export function getImage(src, options) {
```

Comment:
I feel like we might want `width` and `height` here as required parameters.

It's much nicer to write this:
```
<img {...getImage('./my-image.jpg', 512, 256)} alt="my image" />
```

Than this:
```
<img {...getImage('./my-image.jpg', { width: 512, height: 256})} alt="my image" />
```

I do like the explicitness of `width` and `height` being in the options object, but then you end up with `})}` which is quite hard to parse and it just sort of seems like a lot of characters to repeat.